handlers/slot.go

```diff
@@ -511,7 +511,7 @@ func SlotDepositData(w http.ResponseWriter, r *http.Request) {
 			utils.FormatPublicKey(deposit.PublicKey),
 			utils.FormatBalance(deposit.Amount, currency),
 			utils.FormatWithdawalCredentials(deposit.WithdrawalCredentials, true),
-			deposit.Signature,
+			fmt.Sprintf("0x%v", hex.EncodeToString(deposit.Signature)),
```

Comment:
Please use `utils.FormatHash` for this as done, for example, here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/20a7a4ee12dec21e06586566257bbd86618a01aa/handlers/eth2Deposits.go#L87

**Reasoning:**
Navigate to, for example, https://prater.beaconcha.in/validator/0xa370ed2029a9a36c6fa4449a254c9c0e68e01824e5a60be5b9bd1ebe78c79e99b2177112d6008b4f2c41864c5445d2b1#deposits and take a look at the deposit's signature there:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/8cff2b02-d49f-45fd-b95d-64d56317fbc1)
It shows the first and last 4 characters which is reasonable.

If you run you branch on Prater locally and navigate to http://localhost:8081/slot/5598476#deposits though, the same signature is displayed like this though:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/fdf86651-7f93-499e-af64-76907c80b0b7)

---

Furthermore, the font type currently used is wrong (as it isn't monospaced). This should be fixed by using `utils.FormatHash` too.

---

**Note:**
FYI, I have not tested the change I am suggesting, maybe more changes (for example in the according HTML) are required too.

Reply:
Yeap, there were a couple of more things needed in the html side to be taken of.